server/enterprise/metrics/metrics.go

```diff
@@ -1244,6 +1249,42 @@ func New(ps *platform.PlatformService, driver, dataSource string) *MetricsInterf
 	)
 	m.Registry.MustRegister(m.ClientGlobalThreadsLoadDuration)
 
+	m.MobileClientLoadDuration = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: MetricsNamespace,
+			Subsystem: MetricsSubsystemClientsMobileApp,
+			Name:      "mobile_load",
+			Help:      "Duration of the time taken from when a user opens the app and the app finally loads all relevant information (milliseconds)",
+			Buckets:   []float64{1, 1.5, 2, 3, 4, 4.5, 5, 5.5, 6, 7.5, 10, 20, 25, 30},
+		},
+		[]string{"platform", "agent"},
```

Comment:
Do we need the `agent` label here? We only need that for the web app metrics to differentiate different browsers, but we don't really need it for mobile.

We could consider tracking the version of the mobile app instead, but that could also be added later

Reply:
The agent is not really needed, but the way it is implemented right now, the metric verification requires an agent. It is something we can look a bit deeper into.